spec/browsers/apple_podcasts_user_agent_spec.rb

```diff
@@ -158,3 +158,13 @@
   it_behaves_like 'Apple Podcasts with details', '22', 'iOS', 'iOS 12.3.x'
 end
 
+describe "%E3%83%9D%E3%83%83%E3%83%89%E3%82%AD%E3%83%A3%E3%82%B9%E3%83%88/1709.2.2 CFNetwork/1390 Darwin/22.0.0" do 
+  let!(:useragent) { UserAgent.parse("%E3%83%9D%E3%83%83%E3%83%89%E3%82%AD%E3%83%A3%E3%82%B9%E3%83%88/1709.2.2 CFNetwork/1390 Darwin/22.0.0") }
```

Comment:
```suggestion
  let!(:useragent) { UserAgent.parse("%E3%83%9D%E3%83%83%E3%83%89%E3%82%AD%E3%83%A3%E3%82%B9%E3%83%88/1709.2.2 CFNetwork/1390 Darwin/22.0.0") }
  
```
Super nitpick for styling. Just to add a blank line to be consistent with the above.